pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "epicscorelibs", 
-	"swig", 
-	"epicscorelibs_pcas @ git+https://github.com/IsisComputingGroup/epicscorelibs_pcas@main"
```

Comment:
i guess this wants to be pointed at main after https://github.com/ISISComputingGroup/epicscorelibs_pcas/pull/3 is merged?

Reply:
Yes